### PR TITLE
Add dark mode to statics graphs

### PR DESCRIPTION
**Pull request by TheColaber**
Try on https://scratch.mit.edu/statistics/

image

### CODE REVIEW

File: addons/dark-www/experimental_statistics.css [hunk 1]
```diff
@@ -1,5 +1,16 @@
 .box .box-content {
-  /* Charts don't look well on dark background */
-  background: #fff;
+  background: #282828;
   color: #575e75;
```
Comment (mxmou) [inline addons/dark-www/experimental_statistics.css:3]: You can remove this line and line 6.

**Review by mxmou — CHANGES_REQUESTED** [review]
Could you also change the color of the grid lines? Maybe white for the black one and `#333` for the gray ones.

File: addons/dark-www/experimental_statistics.css [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
 }
+text {
+  color: white;
+  fill: currentColor;
+}
+.nvtooltip {
+  background: #000;
```
Comment (mxmou) [inline addons/dark-www/experimental_statistics.css:10]: Use `#282828`

**Comment by TheColaber** [comment]
I think it's a bit hard to see at `#000000`
image
I think `#606060` is better maybe
image

**Review by mxmou — APPROVED** [review]

**Pull request merged**